### PR TITLE
Added missing content for network trust stores plus other edits

### PR DESCRIPTION
**Pull request by lachellel**
see commit messages
Aligned the dates for identifiers and certificates with NIST and FPKI publications
Added trust store for network 
starting on developer guide cleanup and adding issues for additional content

### CODE REVIEW

**Comment by lachellel** [comment]
@grandamp @konklone 

would love some feedback - I'd like to push to additional colleagues for review tomorrow afternoon
I also need to fix the sidebar to make it not show "index" in the collection navigations (index content shows as top level and as sub navigation element)

File: _devconfig/index.md [hunk 1]
```diff
@@ -5,16 +5,18 @@ permalink: /devconfig/
 collection: devconfig
 ---
 
-PIV Developer Guides are designed to help system administrators configure their organization's systems to require PIV authentication. These guidance documents are often highly technical in nature and cover specific step-by-step procedures to configure a system successfully. All completed guidance documents can be found in the Developer Guides dropdown in the navigation bar to the left.
+PIV Developer Guides are to help engineers configure agency applications to require PIV authentication. These guides are intended to be technical and include step-by-step procedures to develop or configure a system, and should be specific to US Federal Government and PIV credentials.
```
Comment (konklone) [inline _devconfig/index.md]: I love this change.

**Comment by konklone** [comment]
These all seem like good, clear changes. I noticed one thing I'm going to open a separate issue about.

**Comment by lachellel** [comment]
@konklone @tkpk 

updated with the change in verbiage for fpki common root cert

Thoughts and suggestions? I'd like to merge if it's okay...

**Comment by konklone** [comment]
:+1: on the changes and merging. I have some suggestions for improving it, but I will submit those in a followup PR.

**Comment by tkpk** [comment]
Sorry for the delayed response.  Looks fine.  Thanks.

**Pull request merged**